salt/utils/__init__.py

```diff
@@ -283,12 +289,10 @@ def get_user():
     '''
     if HAS_PWD:
         return pwd.getpwuid(os.geteuid()).pw_name
+    elif HAS_WIN32:
+        return salt.utils.win_functions.get_current_user()
     else:
-        user_name = win32api.GetUserNameEx(win32api.NameSamCompatible)
-        if user_name[-1] == '$' and win32api.GetUserName() == 'SYSTEM':
-            # Make the system account easier to identify.
-            user_name = 'SYSTEM'
-        return user_name
+        return  # What should I do ?
```

Comment:
Here I would recommend that you `raise salt.exceptions.CommandExecutionError('Required external libraries not found. Need 'pwd' or 'win32api')`